addons/boa/tests/test_tasks.py

```diff
@@ -68,24 +72,7 @@ def test_handle_boa_error(self):
                 output_file_name=self.output_file_name,
                 job_id=self.job_id
             )
-            mock_send_mail.assert_called_with(
-                to_addr=self.user_username,
-                mail=ADDONS_BOA_JOB_FAILURE,
-                fullname=self.user_fullname,
-                code=BoaErrorCode.UNKNOWN,
-                message=self.error_message,
-                query_file_name=self.query_file_name,
-                file_size=self.file_size,
-                max_file_size=boa_settings.MAX_SUBMISSION_SIZE,
-                query_file_full_path=self.file_full_path,
-                output_file_name=self.output_file_name,
-                job_id=self.job_id,
-                max_job_wait_hours=self.max_job_wait_hours,
-                project_url=self.project_url,
-                boa_job_list_url=boa_settings.BOA_JOB_LIST_URL,
-                boa_support_email=boa_settings.BOA_SUPPORT_EMAIL,
-                osf_support_email=osf_settings.OSF_SUPPORT_EMAIL,
-            )
+            self.mock_send_grid.assert_called()
```

Comment:
A bit lazy just to assert this but asserting every single thing in a template is unmaintainable.